src/SFA.DAS.FAA.Web/Controllers/VacanciesController.cs

```diff
@@ -28,7 +28,10 @@ public async Task<IActionResult> Vacancy([FromRoute] GetVacancyDetailsRequest re
 
         var result = await mediator.Send(new GetApprenticeshipVacancyQuery
         {
-            VacancyReference = request.VacancyReference
+            VacancyReference = request.VacancyReference,
+            CandidateId = User.Claims.CandidateId().Equals(Guid.Empty)
```

Comment:
i dont think that the claim not present value is used, so can possibly look at changing the extension to pass null instead

Reply:
Change Claims -> Candidate Id to return null instead of Guid.Empty